pkg/vsphere/compute/placement/ranked_defaults.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package placement
```

Comment:
Are we planning to use these constants elsewhere besides this package? if not I would make them private, if so I would make this `placement-constants` or somethinglike that package wise. so that people do not have to pull the entire package in just for the constants.